db/migrate/20190718152003_add_phone_number_to_personal_details.rb

```diff
@@ -0,0 +1,5 @@
+class AddPhoneNumberToPersonalDetails < ActiveRecord::Migration[5.2]
```

Comment:
Does having the different fields in separate migrations make something easier? It's not a pattern I've seen before.

Reply:
This is because of TDD, I iterated field by field.
Next time we can group the migration by Form to create fewer files.
The way to do it is by rolling back and adding new field every time.